### PR TITLE
Lower Upper East Norfair - Gravity Lava Frames

### PR DESCRIPTION
**Pull request by kjbranch**
Partially addresses #663 
- Lava Dive
- Volcano Room
- Spiky Platforms Tunnel

### CODE REVIEW

File: region/norfair/east.json [hunk 1]
```diff
@@ -5633,6 +5635,7 @@
                       "shinesparkFrames": 33
                     }},
                     {"heatFrames": 500},
+                    {"lavaFrames": 300},
```
Comment (osse101) [inline region/norfair/east.json:5638]: I think its just under 3s.
And the heat frame count is 7s.

File: region/norfair/east.json [hunk 2]
```diff
@@ -5995,7 +6001,14 @@
                   "notable": false,
                   "requires": [
                     "Gravity",
-                    "Morph"
+                    "Morph",
```
Comment (osse101) [inline region/norfair/east.json:6004]: mockball saves about 10 frames.

**Review by osse101 — COMMENTED** [review]
Frame counting lava dive is the worst.

File: region/norfair/east.json (hunk 1 above)
Comment (kjbranch) [reply region/norfair/east.json:5638]: This is including the time to get to the surface, while in the shinespark, right?

Comment (osse101) [reply region/norfair/east.json]: Yeah, I timed it by updating the room timer when I entered the lava and when I exited the lava.

File: region/norfair/east.json [hunk 3]
```diff
@@ -5643,7 +5646,8 @@
                     "Store the shinespark on the last possible pixels of runway.",
                     "Quickly drop to the nearby namihe and damageboost using its flame.",
                     "Hold the damage boost until just before being below the above platform and spark upwards"
-                  ]
+                  ],
+                  "devNote": "FIXME these lava frames are just an approximate number."
```
Comment (osse101) [inline region/norfair/east.json:5650]: Shouldn't need this anymore

**Review by osse101 — APPROVED** [review]

**Pull request merged**